Misc/NEWS.d/next/Library/2022-03-18-13-30-40.bpo-47061.etLHK5.rst

```diff
@@ -0,0 +1,7 @@
+Deprecate the various modules listed by :pep:`594`:
+
+- aifc - asynchat: https://bugs.python.org/issue47022 - asyncore:
+https://bugs.python.org/issue47022 - audioop - cgi - cgitb - chunk - crypt -
+imghdr - msilib - nntplib - nis - ossaudiodev - pipes - smtpd:
+https://bugs.python.org/issue47022 - sndhdr - spwd - sunau - telnetlib - uu
+- xdrlib
```

Comment:
Docs build is failing:

```
Warning, treated as error:
../build/NEWS:59:Bullet list ends without a blank line; unexpected unindent.
make: *** [Makefile:53: build] Error 2
##[error]Bash exited with code '2'.
```

Removing this initial hyphen is one fix:

```suggestion
aifc - asynchat: https://bugs.python.org/issue47022 - asyncore:
https://bugs.python.org/issue47022 - audioop - cgi - cgitb - chunk - crypt -
imghdr - msilib - nntplib - nis - ossaudiodev - pipes - smtpd:
https://bugs.python.org/issue47022 - sndhdr - spwd - sunau - telnetlib - uu
- xdrlib
```

Will produce a NEWS file containing:

```
- bpo-47061: Deprecate the various modules listed by :pep:`594`:

  aifc - asynchat: https://bugs.python.org/issue47022 - asyncore:
  https://bugs.python.org/issue47022 - audioop - cgi - cgitb - chunk - crypt
  - imghdr - msilib - nntplib - nis - ossaudiodev - pipes - smtpd:
  https://bugs.python.org/issue47022 - sndhdr - spwd - sunau - telnetlib -
  uu - xdrlib
```


(Reminder about https://github.com/python/cpython/pull/30419 to upload a generated NEWS file as part of the build.)